src/paper/related_models/paper_model.py

```diff
@@ -820,6 +833,19 @@ class PaperVersion(models.Model):
     message = models.TextField(default=None, null=True, blank=True)
     created_date = models.DateTimeField(auto_now_add=True)
     updated_date = models.DateTimeField(auto_now=True)
+    journal = models.CharField(
+        max_length=32,
+        choices=JOURNAL_CHOICES,
+        null=True,
+        blank=True,
+        help_text="The journal this paper version belongs to",
+    )
+    publication_status = models.CharField(
+        max_length=20,
```

Comment:
Use a text field and avoid arbitrary length constraints.
```suggestion
    publication_status = models.TextField(
```